spec.bs

```diff
@@ -108,6 +108,7 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
       text: dom interface; url: concept-element-dom
       text: accessibility considerations; url: concept-element-accessibility-considerations
       text: represents; url: represents
+      text: is initial about:blank; url: is-initial-about-blank
```

Comment:
Is this needed? I don't see it referenced in this PR.

Reply:
Not entirely sure why this is there, but this was the source of the merge conflict. I'll remove it